header-hero-done

vladmw5 opened the pull request:
Заголовок и герой полностью закончены: html + mob + tab + desk + backdrop. Осталось только модалка

vladmw5 committed:
header-hero-done

Vlad-Kozak committed:
Merge branch 'main' into feature/header-hero

Vlad-Kozak reviewed (changes requested):
1 нужно сделать ховер ссылок в хедере
2 на всех ховерах сделать плавность 
3 кнопку стрелочку в хиро сделать такую же как в секции products 
4 попробуй сделать круг за мороженным в мобилке не обрезанным (от ширины 480 до 767 выглядит не очень)

Vlad-Kozak committed:
Update index.html

vladmw5 committed:
links-buttons-fixed

Links now have transitions and hovers
A button with arrow now is like in products
Two buttons in hero now have transitions
Buy Now button also has transition
